src/test/java/org/swisspush/reststorage/mocks/FailFastVertxFileSystem.java

```diff
@@ -40,6 +51,11 @@ public FileSystem copyRecursive(String s, String s1, boolean b, Handler<AsyncRes
         throw new UnsupportedOperationException(msg);
     }
 
+    @Override
+    public Future<Void> copyRecursive(String from, String to, boolean recursive) {
+        return null;
```

Comment:
Why don't we indicate that this operation is not supported, like we do it with the other methods?
```
        throw new UnsupportedOperationException(msg);
```

Reply:
ok